client/components/layout/main-layout/layoutConfig.ts

```diff
@@ -10,6 +10,13 @@ const buildTreePageRouteTab: Tab = {
   path: '/generate',
 };
 const visualizePageRouteTab: Tab = { tabname: 'Visualize', path: '/output' };
+const cookiePreferencesTab: Tab = {
+  tabname: 'Cookie Preferences',
+  path: '/cookie_preferences',
+};
+const securityTab: Tab = { tabname: 'Security', path: '/security' };
```

Comment:
Consider keeping the naming of your variables consistent with the rest of the document.